src/handler/AdminHandler.ts

```diff
@@ -1,26 +1,34 @@
-import {AdminService} from "../services/AdminService";
+import { AdminService } from "../services/AdminService";
 import JWTs from "../utils/JWTs";
-import {Forbidden, NotFound, UnprocessableEntity} from "http-errors";
+import {
+  Forbidden,
+  NotFound,
+  Unauthorized,
+  UnprocessableEntity,
+} from "http-errors";
 
 const adminService = new AdminService();
 
 export class AdminHandler {
   public async login(username: string, password: string): Promise<string> {
-    if (!await adminService.checkIfUsernameExists(username))
-      throw new Forbidden("Incorrect username or password")
-    if (!await adminService.checkIfPasswordCorrect(username, password))
-      throw new Forbidden("Incorrect username or password")
-    return JWTs.generateAccessToken(username)
+    if (!username || !password)
+      throw new Unauthorized("Missing username or password");
```

Comment:
This StatusCode seems wrong to me.
I know that the status codes have some room for interpretation and this could definitely be one way of doing so.
What do you think @LeoTuet, @quintus-luis?

Reply:
I think its up to interpretation as well.
I would personally go with 401. Because the server understood the request but is refusing to fulfill it due to false credentials therefor the request ist unauthorized to get an jwt.    ([source](https://stackoverflow.com/questions/32752578/whats-the-appropriate-http-status-code-to-return-if-a-user-tries-logging-in-wit))